machine-learning/tutorials/MovieRecommendation/Program.cs

```diff
@@ -81,43 +81,52 @@ public static ITransformer BuildAndTrainModel(MLContext mlcontext, IDataView tra
                 NumberOfIterations = 20,
                 ApproximationRank = 100
             };
+            // </SnippetAddAlgorithm>
```

Comment:
Move this tag down to line 89 or create a new set of tags around the algorithm append.